gpii/node_modules/flowManager/src/SettingsPutHandler.js

```diff
@@ -24,10 +24,11 @@ fluid.defaults("gpii.flowManager.cloudBased.settings.put.handler", {
             funcName: "gpii.flowManager.cloudBased.settings.put.handleRequest",
             args: [
                 "{flowManager}.prefsServerDataSource",
+                "{gpii.flowManager.cloudBased}.authGrantFinder",
```

Comment:
This argument payload is now over the critical mass - please encode it as a struct with named members rather than a sequence of 6 arguments